apps/ledger-live-mobile/src/screens/BleDevicePairingFlow/index.tsx

```diff
@@ -95,12 +101,25 @@ export const BleDevicePairingFlow = ({ navigation, route }: Props) => {
     navigationType = "navigate",
   } = onSuccessNavigateToConfig;
 
+  const recoverConfig: ServicesConfig | null = useFeature(
+    "protectServicesMobile",
+  );
+
   // Makes sure the pairing components are reset when navigating back to this screen
   const keyToReset =
     useIncrementOnNavigationFocusState<Props["navigation"]>(navigation);
 
+  const recoverRestoreFlowURI = usePostOnboardingURI();
+
   const onPairingSuccess = useCallback(
     (device: Device) => {
+      if (recoverConfig?.enabled && recoverRestoreFlowURI && isRecoverFlow) {
+        Linking.canOpenURL(recoverRestoreFlowURI).then(() =>
```

Comment:
canOpenURL resolves in a boolean which says whether or not it can open the URL, so you should use it, otherwise it has no use